components/dropindicator/stories/template.js

```diff
@@ -1,21 +1,30 @@
 import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map.js';
-// import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { styleMap } from 'lit-html/directives/style-map.js';
 
 import '../index.css';
 import '../skin.css';
 
 export const Template = ({
   rootClass = "spectrum-DropIndicator",
   customClasses = [],
-  size = "m",
-  // ...globals
+  direction = "vertical",
+  length,
 }) => {
+  const styles = {
+    height: direction == 'vertical' ? length + "px" : "",
+    width: direction == 'horizontal' ? length + "px" : "",
+  };
+
   return html`
-    <div class=${classMap({
-      [rootClass]: true,
-      [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
-    ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
-  })}></div>
+    <div style="display: flex; align-items: center">
```

Comment:
```suggestion
    <div style="display: flex; align-items: center">
```

Can you pull this customization out? If we end up needing to embed the component in another one, we just want the template to contain a display tool.  If you like, you can add this to the story:

```
Default.parameters = {
  layout: 'centered'
};
```

Reply:
I was trying to recreate the markup on the docs site but let me see if it works without!